app/javascript/dashboard/components/ChatList.vue

```diff
@@ -321,6 +323,9 @@ export default {
     hasAppliedFiltersOrActiveFolders() {
       return this.hasAppliedFilters || this.hasActiveFolders;
     },
+    savedSortByFilter() {
+      return this.uiSettings?.conversations_filter_by;
+    },
```

Comment:
If this computed value is not used in other places, also that `setSavedSortByFilter` is only triggered at `created`, we can get rid of this method and access it in `setSavedSortByFilter` directly